### PR TITLE
[manila-csi-plugin] Add extra labels to k8s resources

### LINKED ISSUE #1786: [Manila-CSI-plugin] Add extra labels to k8s resources in helm chart
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
kind feature

**What happened**:
I reckon that having the possibility to add optoinal extra labels to the K8s resources would add flexibility to the Helm chart. 

**What you expected to happen**:
It is possible to add arbitrary labels to the k8s resources in an optional way.

**How to reproduce it**:
N/A

**Anything else we need to know?**:
I've already written the code for that, I will submit it so you can evaluate and approve if you consider it appropriate

**Environment**:
- openstack-cloud-controller-manager(or other related binary) version: N/A
- OpenStack version: N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by JPRbrs**
Extra labels are optional and applied to all resources already sporting labels

**What this PR does / why we need it**:
Having the possibility of adding optional extra labels to K8s resources would make the Helm chart more flexible.

**Which issue this PR fixes(if applicable)**:
fixes #1786

**Special notes for reviewers**:
I've tested it setting extraValues.enabled to true and adding some dummy labels as well as with no values and extraValues.enabled set to false. Both times I've run helm3 template . 

**Release note**:
```release-note
[Manila-CSI-plugin]
Adding optional extra labels to be set in .Values.extraLabels.labels. Enable the usage of extra labels setting .Values.extraLabels.enable to true
```

### CODE REVIEW

File: charts/manila-csi-plugin/values.yaml
Comment (gman0) [inline charts/manila-csi-plugin/values.yaml:19]: Is `enabled` needed? Won't just leaving the default value at `{}` do the job?

**Comment by gman0** [comment]
@JPRbrs thank you for your PR! One comment in-line, otherwise lgtm. Also please add `[manila-csi-plugin] ` prefix to the release note.

I think it would be good if we kept Cinder's and Manila's charts more or less in sync. Would you mind pushing a follow up PR to add this `extraLabels` to Cinder helm chart too?

File: charts/manila-csi-plugin/values.yaml
Comment (JPRbrs) [reply charts/manila-csi-plugin/values.yaml:19]: you're right, I'l change it, thanks!


> @JPRbrs thank you for your PR! One comment in-line, otherwise lgtm. Also please add `[manila-csi-plugin] ` prefix to the release note.
> 
> I think it would be good if we kept Cinder's and Manila's charts more or less in sync. Would you mind pushing a follow up PR to add this `extraLabels` to Cinder helm chart too?

Definitely, no problem at all!

**Comment by JPRbrs** [comment]
comments addressed @gman0.

Thanks!

**Review by gman0 — APPROVED** [review]
All good, @JPRbrs. Thank you!

/lgtm
/approve

**Comment by gman0** [comment]
@jichenjc can you please /approve if this looks ok to you?

**Comment by gman0** [comment]
@JPRbrs I believe new charts are only pushed along with regular CPO releases -- unless it's a bugfix patch release.

I'm not CPO repo maintainer but I'm not sure this PR is eligible for a bugfix release. Maybe others and @lingxiankong can tell you more?

**Comment by JPRbrs** [comment]
Many thanks for your quick response! I'll wait to see what other people say, thanks again!

**Pull request merged**
